tests/fixtures.py

```diff
@@ -523,6 +523,23 @@ def inject_set_hvac_schedules(mocked_responses: aioresponses) -> str:
         "vehicle_action/hvac-schedule.schedules.json",
     )
 
+def inject_start_horn(mocked_responses: aioresponses) -> str:
```

Comment:
Pre-commit says there should be two lines between functions:
```suggestion
    )


def inject_start_horn(mocked_responses: aioresponses) -> str:
```